linchpin/provision/roles/aws/tasks/teardown_resource_group.yml

```diff
@@ -60,13 +60,26 @@
     topo_output_resources: "{{ topo_output_rundb.output[0]['resources'] }}"
   when: not generate_resources
 
+- name: "print data"
+  debug:
+    msg: "{{ topo_output_resources }}"
+
+#- name: "filter topo_output_resources to only include ec2 resource definitions"
```

Comment:
Is there any reason to keep these commented code . ??
@14rcole

Reply:
Hm, I didn't realize that had been commented out.  Well, if it works I'll remove it